File: Java8-JavaFX/src/sevenguis/circledrawer/Circle.java
package sevenguis.circledrawer;

public class Circle {

    private int x, y, d;

    public Circle(int x, int y) {
        this.x = x;
        this.y = y;
        d = 30;
    }

    public int getX() {
        return x;
    }

    public void setX(int x) {
        this.x = x;
    }

    public int getY() {
        return y;
    }

    public void setY(int y) {
        this.y = y;
    }

    public int getDiameter() {
        return d;
    }

    public void setDiameter(int d) {
        this.d = d;
    }

}

File: Qt5/circledrawer/source/circle.cpp
#include "circle.h"

Circle::Circle(int x, int y)
{
    this->x = x;
    this->y = y;

    diameter = 30;
}

int Circle::getX()
{
    return x;
}

void Circle::setX(int x)
{
    this->x = x;
}

int Circle::getY()
{
    return y;
}

void Circle::setY(int y)
{
    this->y = y;
}

int Circle::getDiameter()
{
    return diameter;
}

void Circle::setDiameter(int diameter)
{
    this->diameter = diameter;
}

Circle::~Circle()
{

}
